server/emails/index.js

```diff
@@ -0,0 +1,21 @@
+import sendVerificationEmail from "./verification/email";
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
remove this;it is throwing an error because you should `export default emailVerification` at the end of the file